appleseed-Installer/VersionSelectDlg.wxs

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--//**********************************************************************/
+// Copyright (c) 2021 appleseed.
+// All rights reserved.
+// 
+//**************************************************************************/
+// DESCRIPTION: Contains the Dll Entry stuff
+// AUTHOR: Omid Ghotbi (TAO) omid.ghotbi@gmail.com www.3dfine.com
+//***************************************************************************/-->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    
+    <UI Id="VersionSelectDialogUI">
+
+      <Property Id="INSTALLSELECT2017" Secure="yes" />
+      <Property Id="INSTALLSELECT2018" Secure="yes" />
+      <Property Id="INSTALLSELECT2019" Secure="yes" />
+      <Property Id="INSTALLSELECT2020" Secure="yes" />
+      <Property Id="INSTALLSELECT2021" Secure="yes" />
+
+      <DialogRef Id="PathSelectDialog"/>
+
+      <Dialog Id="VersionSelectDlg" Width="555" Height="416" Title="appleseed : 3dsmax selection">
+
+        <Control Id="background" Type="Bitmap" X="0" Y="0" Width="555" Height="416" Text="bgImage" TabSkip="no" FixedSize="yes" />
+
+        <Control Id="SelectText"  Type="Text" X="270" Y="110" Width="300" Height="22" Transparent="yes" Text="{\TahomaHeader}Select 3dsmax version" />
+        <Control Id="selectExplanationText" X="230" Y="150" Width="295" Height="100" NoWrap="no" RightAligned="no" Transparent="yes" Type="Text" Text="{\TahomaNormal}Select the 3dsmax version you want to install the plugin. Click on the Cancel button to exit." />
```

Comment:
Please select the Autodesk® 3ds Max® version for which you would like to install appleseed, or click Cancel to exit.

Reply:
done.

Reply:
done.